ARMeilleure/Decoders/IOpCode32Simd.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ARMeilleure.Decoders
+{
+    interface IOpCode32Simd : IOpCode32, IOpCodeSimd
```

Comment:
```suggestion
    interface IOpCode32Simd : IOpCode32, IOpCodeSimd { }
```
maybe put the bracket on the same line could be better?